albumentations/augmentations/transforms.py

```diff
@@ -952,16 +953,11 @@ def get_transform_init_args_names(self):
         return "height", "width", "scale", "ratio", "interpolation"
 
 
-class RandomSizedBBoxSafeCrop(DualTransform):
+class BBoxSafeRandomCrop(DualTransform):
     """Crop a random part of the input and rescale it to some size without loss of bboxes.
```

Comment:
Update docstring here: remove "and rescale..."

Reply:
Updated this in c0621eee7c3e03df3343c4eae5b7a571cb9cb6cb!